src/main.rs

```diff
@@ -149,14 +150,20 @@ fn main() {
     options.output_style = match matches.value_of("style") {
         Some("full") => OutputStyleOption::Full,
         Some("basic") => OutputStyleOption::Basic,
+        Some("nocolor") => OutputStyleOption::NoColor,
         _ => if atty::is(Stream::Stdout) {
             OutputStyleOption::Full
         } else {
             OutputStyleOption::Basic
         },
     };
 
-    if options.output_style == OutputStyleOption::Basic {
+    // We default Windows to NoColor if full had been specified.
+    if cfg!(windows) && options.output_style == OutputStyleOption::Full {
+        options.output_style = OutputStyleOption::NoColor;
```

Comment:
If implemented in this way, there is no way to change to `Full` on Windows, right?

Reply:
That’s correct, on Windows, full is essentially NoColor. I felt that it would just improve consistency across shells on the platform.

Reply:
Well, there’s nothing about the colours that don’t work on cmd or powershell, but they are just not properly visible on the default powershell background. I think we have two straightforward options: either we make full->full and auto->nocolor or change the current colour choices to properly be visible on the default powershell background. After thinking about this more the latter seems the most reasonable. My screenshot in the PR for enabling the progress bar shows the problem areas

Reply:
Okay, right. I'll merge this and if we want to enable color support on Windows, we should first update the colors.